src/index.html

```diff
@@ -8,6 +8,52 @@
     <link rel="stylesheet" href="./style.css">
   </head>
   <body>
-    <h1>Stars</h1>
+    <div class="stars stars--0">
+      <div class="stars__star"></div>
+      <div class="stars__star"></div>
+      <div class="stars__star"></div>
+      <div class="stars__star"></div>
+      <div class="stars__star"></div>
+    </div>
+
+    <div class="stars stars--1">
+      <div class="stars__star stars__star--active"></div>
```

Comment:
```suggestion
    <div class="stars stars--1">
      <div class="stars__star"></div>
```
Stars elements should have only one class
![image](https://github.com/mate-academy/layout_stars/assets/96409563/6cbd7e9b-c089-4a98-90c2-9e19ec7818a9)

Reply:
Gotcha!
Maked backgroung with nth-child selectors